tests/src/Command/ListTablesCommandTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Db\Migration\Tests\Command;
+
+use PHPUnit\Framework\TestCase;
+use Psr\Container\ContainerInterface;
+use Symfony\Component\Console\Tester\CommandTester;
+use Yiisoft\Db\Connection\ConnectionInterface;
+use Yiisoft\Yii\Console\ExitCode;
+use Yiisoft\Yii\Db\Migration\Command\ListTablesCommand;
+use Yiisoft\Yii\Db\Migration\Migrator;
+use Yiisoft\Yii\Db\Migration\Service\Database\ListTablesService;
+use Yiisoft\Yii\Db\Migration\Service\MigrationService;
+use Yiisoft\Yii\Db\Migration\Tests\Support\PostgreSqlHelper;
+
+final class ListTablesCommandTest extends TestCase
+{
+    public function testSeveralSchemas(): void
+    {
+        $container = PostgreSqlHelper::createContainer();
+
+        PostgreSqlHelper::clearDatabase($container);
```

Comment:
Because it takes `yiisoft / db` doesn't need this magic.

Reply:
PostgreSQL need for test list tables with several schemas.